Assets/Scripts/AvatarMoving.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AvatarMoving : MonoBehaviour
+{
+    private float speed = 100;
+
+    private Rigidbody rb;
+
+    void Start()
+    {
+        rb = GetComponent<Rigidbody>();
+    }
+
+    // Update is called once per frame
+    void Update()
```

Comment:
```suggestion
    void Update()
    {
        float h = Input.GetAxis("Horizontal");
        float v = Input.GetAxis("Vertical");

        Vector3 directionVector = new Vector3(h, 0, v);
        rb.velocity = directionVector * speed;
    }
```
Вынести в отдельный метод.